initrd/bin/kexec-iso-init

```diff
@@ -27,7 +27,7 @@ mount -t iso9660 -o loop $MOUNTED_ISO_PATH /boot \
 	|| die '$MOUNTED_ISO_PATH: Unable to mount /boot'
 
 DEV_UUID=`blkid $DEV | tail -1 | tr " " "\n" | grep UUID | cut -d\" -f2`
-ADD="fromiso=/dev/disk/by-uuid/$DEV_UUID/$ISO_PATH img_dev=/dev/disk/by-uuid/$DEV_UUID iso-scan/filename=/${ISO_PATH} img_loop=$ISO_PATH"
+ADD="fromiso=/dev/disk/by-uuid/$DEV_UUID/$ISO_PATH img_dev=/dev/disk/by-uuid/$DEV_UUID iso-scan/filename=/${ISO_PATH} img_loop=$ISO_PATH iso=$DEV_UUID/$ISO_PATH"
```

Comment:
Wish there was a bit more of a standard here so we didn't have to spray all the variants we know about :sweat:   But it looks fine for our purposes

Reply:
Will add reference to webboot as comment prior of section. Since this is passed as kernel options down to user land init, passing them all is pretty harmless